build.yaml

```diff
@@ -3008,8 +3007,7 @@ steps:
               --instafail \
               --durations=50 \
               --timeout=360 \
-              /io/test/hailtop/batch/ \
-              /io/test/hailtop/hailctl/batch/
+              /io/test/hailtop/batch/
```

Comment:
I'm not sure if this is meaningful but I think these tests were getting run twice. Here and wherever the rest of the `hailctl` tests are run